src/(components)/SignUpForm.tsx

```diff
@@ -0,0 +1,76 @@
+// In a new file called SignUpForm.tsx
+import React, { useState } from 'react'
+
+interface FormData {
+  username: string
+  password: string
+  rememberMe: boolean
+}
+
+const SignUpForm = () => {
+  // Use the useState hook to manage the form data
+  const [formData, setFormData] = useState<FormData>({
+    username: '',
+    password: '',
+    rememberMe: false,
```

Comment:
Would be a good place to persist the session and remember the user's device - though as of now I'm not sure how to